### DISCUSSION TITLE
Allow to edit post excerpt through Recent Posts

### DISCUSSION THREAD

**Discussion by htmgarcia** [publishpress/publishpress-blocks#1201] [Feature Requests]
It would be really useful to allow to edit/add a post excerpt directly through Recent Posts to preview the end result without leaving the screen.
<img width="767" alt="Screen Shot 2021-04-09 at 15 26 21" src="https://user-images.githubusercontent.com/4999794/114236977-01b0ef80-9948-11eb-9b1b-057f73b7af77.png">

**Comment by htmgarcia** [comment]
@stevejburge I think this feature should go in Pro.

**Comment by htmgarcia** [comment]
@contactashish13 
1. How possible is to add this feature? Basically is edit the post excerpt inside the block.
2. It can be done as a feature available in Pro version only?

**Comment by contactashish13** [comment]
@htmgarcia 

1. stupid question but should the excerpt be saved back into the post or is it only for that instance of content display?
2. if we do offer this, won't users want control over aspects of the post as well e.g. featured image or title?

**Comment by stevejburge** [comment]
@contactashish13 Let's skip this one for now. We're going to focus more on the tasks that involve block management (scheduling, access control etc)
